### DISCUSSION TITLE
Actions Stuck in Requested

### DISCUSSION THREAD

**Discussion by oconnell-fastly** [StackStorm/st2#6263] [Operations] [answered]
I have a new ST2 deployment (using stack-ha helm chart with private repos).  I am able to run actions but they just go into "requested" and hang.   I thought it was something with RabbitMQ but I can authenticate using creds in /etc/st2/st2.docker.conf file.  I know there is a rudimentary diag of how things flow in ST2 on the website, but it would be great to get a granular understanding of the flow (i.e. Is it getting stuck on the scheduler? workflow? etc.). It must be getting to st2actionrunner since the code is being executed.

**Comment by fdrab** [comment, marked as answer]
does your REDIS connection work properly? I've had this same issue when I tried to enable REDIS SSL. Execution was being picked by an actionrunner, but connection to REDIS was failing. What do you see in the deploy/$name-st2actionrunner logs?

**Reply by oconnell-fastly** [reply]
Thanks!  That looks like what the problem was.  I deployed the redis Helm chart that came with stackstorm-ha but for some reason there was no master and thus writes were failing.   After killing sentinel on all the nodes , forcing one of the redises to Master and hard coding the config it looks like it is working.  Appreciate it!
